bundles/AdminBundle/Resources/public/js/pimcore/object/tags/manyToManyRelation.js

```diff
@@ -100,6 +101,15 @@ pimcore.object.tags.manyToManyRelation = Class.create(pimcore.object.tags.abstra
         };
     },
 
+    getRelationFilterCondition: function (editor) {
```

Comment:
@dvesh3 here is the place where object/asset/document id is bundled with it's type as a reply to https://github.com/pimcore/pimcore/pull/10022#discussion_r719203448